AndroidSDKTests/src/test/java/com/leanplum/LeanplumTest.java

```diff
@@ -1123,6 +1123,29 @@ public void testSyncResources() {
     assertTrue((Boolean) TestClassUtil.getField(FileManager.class, "isInitialized"));
   }
 
+  /**
+   * Test that a valid version is checked in.
```

Comment:
Glad you added these tests, but i'm wondering if this file is in fact the best place for these tests.

Reply:
You caught me. I'll make a new file.